source/opt/loop_unroller.cpp

```diff
@@ -842,19 +851,21 @@ void LoopUnrollerUtilsImpl::AssignNewResultIds(BasicBlock* basic_block) {
   }
 }
 
-// For all instructions in |basic_block| check if the operands used are from a
-// copied instruction and if so swap out the operand for the copy of it.
-void LoopUnrollerUtilsImpl::RemapOperands(BasicBlock* basic_block) {
-  for (Instruction& inst : *basic_block) {
-    auto remap_operands_to_new_ids = [this](uint32_t* id) {
-      auto itr = state_.new_inst.find(*id);
+void LoopUnrollerUtilsImpl::RemapOperands(Instruction* inst) {
```

Comment:
Can you keep a variant of the original comment?  The new version doesn't have any documentation.

Reply:
The code standards that we use want the comments describing the function just before the declaration, not the definition.  The declaration at line 255 already has the comments, so the ones I removed are redundant.